src/visual.ts

```diff
@@ -1724,66 +1723,66 @@ export class Visual implements IVisual {
         if (!this.popupHeight || !d || !d.popupInfo || (this.animationIsPlaying() && !d.selected)) {
             return false;
         }
-        return d.selected;
+        return d.selected || d.highlight;
     }
 
     public animationIsPlaying(): boolean {
         return this.animationHandler.isPlaying;
     }
 
-    /* tslint:disable:max-func-body-length */
+    // eslint-disable-next-line max-lines-per-function
     private drawTooltips(data: ChartData): void {
-        let xScale: LinearScale = <LinearScale>data.xScale,
+        const xScale: LinearScale = <LinearScale>data.xScale,
             yScales: LinearScale[] = <LinearScale[]>data.yScales,
             node: ClassAndSelector = Visual.Tooltip,
             nodeParent: ClassAndSelector = Visual.TooltipContainer,
-            width: number = this.data.settings.popup.width,
-            height: number = this.data.settings.popup.height,
+            width: number = this.data.settings.popup.width.value,
+            height: number = this.data.settings.popup.height.value,
             marginTop: number = Visual.DefaultTooltipSettings.marginTop,
-            showTimeDisplayProperty: string = this.data.settings.popup.showTime ? "inherit" : "none",
-            showTitleDisplayProperty: string = this.data.settings.popup.showTitle ? "inherit" : "none";
+            showTimeDisplayProperty: string = this.data.settings.popup.showTime.value ? "inherit" : "none",
+            showTitleDisplayProperty: string = this.data.settings.popup.showTitle.value ? "inherit" : "none";
```

Comment:
Usually variables that starts with 'show' are boolean. Let's rename it just to timeDisplayProperty

Reply:
It is `boolean`, so I don't see the reason to rename it.

Reply:
It's a string `: string`